proto/regen/ecocredit/v1/query.proto

```diff
@@ -56,6 +56,14 @@ service Query {
     option (google.api.http).get = "/regen/ecocredit/v1/projects/{project_id}";
   }
 
+  // ProjectsByAdmin queries for all projects by admin with
+  // pagination.
+  rpc ProjectsByAdmin(QueryProjectsByAdminRequest)
+      returns (QueryProjectsByAdminResponse) {
+    option (google.api.http).get =
+        "/regen/ecocredit/v1/projects/{admin}";
```

Comment:
this will collide with the above query `"/regen/ecocredit/v1/projects/{project_id}";`. we should probably do 

```
"/regen/ecocredit/v1/projects/by-admin/{admin}";
```